### ISSUE TITLE
[Feature Request] Option to Disable Toasts

### ISSUE THREAD

**Issue by KxttyKxt** [isXander/Controlify#367]
### Is your feature request related to a problem? Please describe.
It is not a problem necessarily, but I have a Dualsense controller that I put through DS4Windows for emulation and kb+m purposes. When Minecraft is the focused window, it auto-switches to a Dualshock 4 controller; when the window is unfocused, it reverts to my kb+m controls. The "problem" is that, when alt-tabbing/leaving/coming back to the Minecraft window, **I get two system toasts about a disconnected and reconnected controller.**

### Describe the solution you'd like
I would like a config option to disable these toasts. As it stands, I have to disable all system toasts (through a setting in Fabulously Optimized), and I still hear the vwoosh sound.

### Describe alternatives you've considered
I've tried just not emulating the controller at all, so that way it stays constantly connected. However, my laptop interprets the Dualsense's controls incorrectly, and the only fix I've found is running it through DS4Windows.

### Additional context
I know my problem might be pretty niche, but it would be a small QoL change for the best controller mod I've seen to date. Thanks for all you do.
![image](https://github.com/isXander/Controlify/assets/56617952/a01a2db6-7a41-4927-ba30-840817c15689)
![image](https://github.com/isXander/Controlify/assets/56617952/a7c8f5fe-e5b8-4fd2-90fe-1b6b0c9ed6ff)

**Comment by isXander** [comment]
FYI controlify does not have any need for DS4Windows, it works natively with PS controllers.

**Comment by KxttyKxt** [comment]
Yes. Without DS4Windows it simply reads wrong. (Stick drift, weird input lag when holding a trigger or control stick...)

That's completely aside though. Just a way to turn off toasts might be nice. 

Also, I just remembered, when it picks up a "new" controller, it deletes all the text you have open (chat, command block, sign, etc.).
